doc/experimental/phase1/phase1.md

```diff
@@ -157,6 +157,18 @@ helm get values -n verrazzano-ingress-nginx ingress-controller > overrides.yaml
 sed -i '1d' overrides.yaml
 sed -i '/digest:/d' overrides.yaml
 sed -i '/image:/,+2d' overrides.yaml
+cat > extra-configmap.yaml <<EOF
+  extraConfigMaps:
+  - name: ingress-controller-ingress-nginx-defaultbackend-custom-error-pages
+    data:
+EOF
+kubectl get ConfigMap -n verrazzano-ingress-nginx ingress-controller-ingress-nginx-defaultbackend-custom-error-pages  -o jsonpath={.data} | yq -P > data.yaml
+while IFS= read -r line; do
```

Comment:
Is it possible to do this with the `load` family of operations in `yq`?

Reply:
I can experiment with that and can issue a new PR if that approach works out better.